compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/ComposeUiFlags.kt

```diff
@@ -86,6 +86,13 @@ object ComposeUiFlags {
      */
     @Suppress("MutableBareField") @JvmField var isViewFocusFixEnabled: Boolean = true
 
+    /**
+     * When an embedded view that is focused is removed from the hierarchy, it triggers a
+     * requestFocus() which tries to re-assign focus before the previous composition is complete.
+     * This flag enables a fix for this issue.
+     */
+    @Suppress("MutableBareField") @JvmField var isRemoveFocusedViewFixEnabled: Boolean = true
```

Comment:
We need to re-test interop focus stuff with and without this flag

Reply:
Checked - it works